verity/src/helm/templates/ingress.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if not .Values.ingress.targetGroupARNarn -}}
```

Comment:
this condition doesn't look well. In order to have backward compatibility (with `Ingress`) it would be better to have some explicit variable that could have only one of several values (like ENUM).
For example
```
ingressResource = 'TargetGroupBinding' # possible values are 'TargetGroupBinding'/'Ingress'
```

But now we use `.Values.ingress.targetGroupARNarn` variable as a condition variable for `Ingres`. But, in fact, this variable is not for `Ingress` at all. It is used only for `TargetGroupBinding` resource. I'm afraid that using this variable here will be source of bugs in future.

Reply:
as it was discussed we have now only 2 choices and  `.Values.ingress.targetGroupARNarn` can act as a switch, so keeping the current state for now